packages/design-system/src/components/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Button } from '../Button';
+import Alert from './Alert';
+
+const meta: Meta<typeof Alert> = {
+  component: Alert,
+  parameters: {
+    docs: {
+      controls: { sort: ['alpha'] },
```

Comment:
I like the idea of sorting our props tables alphabetically. I think this setting can be added globally if we go down this route.

Reply:
That's cool